src/SALib/analyze/morris.py

```diff
@@ -319,11 +352,25 @@ def _compute_elementary_effects(
         Number of points in a trajectory
     delta: float
         Scaling factor computed from `num_levels`
+    scaling: bool
+        True if the elementary effects should be in scaled as in [1]
 
     Returns
     ---------
     elementary_effects : np.array
         Elementary effects for each parameter
+
+    References
+    ----------
+    1. Sin and Gearney (2009)
+       Improving the Morris Method for Sensitivity Analysis
+       by Scaling the Elementary Effects.
+       19th European Symposium on Computer Aided Process Engineering ESCAPE19:925-930
+    2. Moret et al. (2017)
```

Comment:
From memory, a space between the two list items are needed for the list to render correctly in the deployed documentation.